src/services/GasEstimator.ts

```diff
@@ -0,0 +1,200 @@
+import { BlockWithTransactions, FeeData, TransactionResponse } from '@ethersproject/abstract-provider';
```

Comment:
did you copied it from`pegasus` or `sanctuary`?

Reply:
Sanctuary

Reply:
good, I think this is latest version